lms/templates/courseware/accordion.html

```diff
@@ -40,9 +40,12 @@
                 ## a status icon
                 <%
                     if section.get('due') is None:
-                        data_string = ''
+                        data_string = _(section['format'])
```

Comment:
Trying to translate this string doesn't make sense. The "format" is specified by course staff in Studio. They don't expect it to be translated, and translation would only work if the string had been scraped before.

In general, translating of a variable is not a good idea (there are a couple of exceptions, but you should always think carefully before translating a variable instead of a literal string).